src/yb/util/env_posix.cc

```diff
@@ -1575,5 +1576,26 @@ std::unique_ptr<Env> Env::NewDefaultEnv(std::unique_ptr<FileFactory> file_factor
   return std::make_unique<PosixEnv>(std::move(file_factory));
 }
 
+extern "C" {
```

Comment:
shouldn't need this forward declaraion of errno if you include `<errno.h>` or `<cerrno>`

Reply:
Removed.